src/components/search-bar/search-bar.styles.ts

```diff
@@ -0,0 +1,50 @@
+import { styled as createStyled, type Theme } from 'baseui';
+import type { ButtonOverrides } from 'baseui/button';
+import { type StyleObject } from 'styletron-react';
+
+export const styled = {
```

Comment:
we have media queries for mobile support already existing v4 why not using them ?